index.js

```diff
@@ -60,13 +60,35 @@ animalToRemove.remove()
 
 /***** Deliverable 1 *****/
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+header.addEventListener('click', function(event){
+    toggleColor(event.target)
+
+})
+
 /***** Deliverable 2 *****/
+const button = document.querySelector('button.like-button')
+button.addEventListener('click', function() {
+    const nowlikes = parseInt(likes.textContent)
+    likes.textContent = `${nowlikes + 1} Likes`
+})
+
+
 
 /***** Deliverable 3 *****/
+
+const newSightingfrom = document.querySelector('form#new-animal-sighting-form')
+newSightingfrom.addEventListener('submit', function (event) {
+    event.preventDefault()
+
+    const species = event.target[0].value
+    const video = event.target[1].value
+    const photo = event.target[2].value
+    const description = event.target[4].value
+})
```

Comment:
good work so far! next steps to completing this deliverable would be to:
1. compiling the values that you pulled out of the input fields into an object, `newSightingObject`
2. rendering them onto the page using `renderAnimalSightingPost` that was provided to us
3. update the original array of traveler sightings by adding the new sighting object into the array: `traveler.animalSightings.push(newSightingObject)`

```
    const lastIndex = traveler.animalSightings.length - 1
    const newSightingObject = {
        id: traveler.animalSightings[lastIndex].id + 1,
        travelerId: 1,
        species: species,
        photo: photo,
        link: video,
        description: description
    }
    traveler.animalSightings.push(newSightingObject)
    renderAnimalSightingPost(newSightingObject)
```